utils/src/lib.rs

```diff
@@ -18,6 +18,8 @@ pub mod tagged_signing;
 #[cfg(feature = "time")]
 pub mod time;
 
+pub mod stream;
```

Comment:
Shouldn't we add a feature such that we don't pull unnecessary dependencies when we depend on this crate and we don't need anything on the stream module?

Reply:
Done, although I doubt its usefulness. All of our crates depend on `futures` directly or indirectly.